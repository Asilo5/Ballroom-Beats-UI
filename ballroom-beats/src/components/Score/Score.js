import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export const Score = () => {
  return ( 
    <View style={styles.container}>
      <Text>Score</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'white',
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export default Score;