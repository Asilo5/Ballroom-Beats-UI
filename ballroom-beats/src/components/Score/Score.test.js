import React from 'react';
import renderer from 'react-test-renderer';
import Score from '../Score/Score';

describe('Score', () => {

   it('should match snapshot', () => {
    const score = renderer.create(<Score />).toJSON();
    expect(score).toMatchSnapshot();
   });

});