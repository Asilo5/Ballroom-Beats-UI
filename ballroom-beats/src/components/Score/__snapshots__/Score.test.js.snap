// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Score
  </Text>
</View>
`;
